### PR TITLE
[test] Add specific tag to checks that test Cray PE

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix UES-572

### CODE REVIEW

File: cscs-checks/prgenv/helloworld.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def __init__(self, lang, linkage):
         self.sourcepath += '_openmp.' + lang
         self.descr += ' OpenMP ' + str.capitalize(linkage)
         self.prgenv_flags = {
-            'PrgEnv-cray': self.cray_omp_flags(lang),
+            'PrgEnv-cray': ['-homp' if lang == 'F90' else '-fopenmp'],
```
Comment (vkarak) [inline cscs-checks/prgenv/helloworld.py]: Why don't you use the `cray_omp_flags`? It is supposed to do what you are doing here.

File: cscs-checks/prgenv/ssh_environ_check.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ def __init__(self):
         self.valid_systems = ['daint:login', 'dom:login']
         self.valid_prog_environs = ['PrgEnv-cray']
         reference = {
-            'CRAY_CPU_TARGET': 'haswell',
+            'CRAY_CPU_TARGET': ('haswell', 'broadwell'),
```
Comment (vkarak) [inline cscs-checks/prgenv/ssh_environ_check.py]: I can't understand how this works.

**Comment by vkarak** [comment]
@jgphpc Can you answer my  questions here? This is PR is ready  to  go otherwise.

File: cscs-checks/prgenv/helloworld.py (hunk 1 above)
Comment (jgphpc) [reply cscs-checks/prgenv/helloworld.py]: i removed the function, it's easier to read inlined

File: cscs-checks/prgenv/ssh_environ_check.py (hunk 2 above)
Comment (jgphpc) [reply cscs-checks/prgenv/ssh_environ_check.py]: indeed. i'll make a separate PR.

**Comment by vkarak** [comment]
@jenkins-cscs retry none

**Pull request merged**